### ISSUE TITLE
Enable ClusterJob for hyperparameter search

### ISSUE THREAD

**Issue by jiwoncpark** [jiwoncpark/fast-forward#1]
- `wr`
-  `dr`
- learning rate (`lr`) 
- gradient clipping value (visualize feature maps to decide on range)
